scaletest/harness/results_test.go

```diff
@@ -45,26 +67,81 @@ func Test_Results(t *testing.T) {
 		ElapsedMS: 1000,
 	}
 
-	expected := `
+	wantText := `
 == FAIL: test-0/0
 
 	Error: test-0/0 error
 
 	Log:
 		test-0/0 log line 1
 
+== FAIL: test-0/2
+
+	Error: test-0/2 error
+
+	Log:
+		test-0/2 log line 1
+
 
 Test results:
 	Pass:  8
 	Fail:  2
 	Total: 10
 
 	Total duration: 1s
-	Avg. duration:  200ms
+	Avg. duration:  300ms
+`
+	wantJSON := `{
+	"total_runs": 10,
+	"total_pass": 8,
+	"total_fail": 2,
+	"elapsed": "1s",
+	"elapsed_ms": 1000,
+	"runs": {
+		"test-0/0": {
+			"full_id": "test-0/0",
+			"test_name": "test-0",
+			"id": "0",
+			"logs": "test-0/0 log line 1\ntest-0/0 log line 2",
+			"started_at": "2023-10-05T12:03:56.395813665Z",
+			"duration": "1s",
+			"duration_ms": 1000,
+			"error": "test-0/0 error:\n    github.com/coder/coder/v2/scaletest/harness_test.Test_Results\n        /home/coder/src/coder/coder/scaletest/harness/results_test.go:40"
+		},
+		"test-0/1": {
+			"full_id": "test-0/1",
+			"test_name": "test-0",
+			"id": "1",
+			"logs": "test-0/1 log line 1\ntest-0/1 log line 2",
+			"started_at": "2023-10-05T12:03:56.728813665Z",
+			"duration": "1s",
+			"duration_ms": 1000,
+			"error": "\u003cnil\u003e"
+		},
+		"test-0/2": {
+			"full_id": "test-0/2",
+			"test_name": "test-0",
+			"id": "2",
+			"logs": "test-0/2 log line 1\ntest-0/2 log line 2",
+			"started_at": "2023-10-05T12:03:57.061813665Z",
+			"duration": "1s",
+			"duration_ms": 1000,
+			"error": "test-0/2 error"
```

Comment:
Review: This is a simple error (but not `{}`) because we don't implement unwrapping on `testError`.